api/turing/api/appcontext.go

```diff
@@ -128,6 +131,7 @@ func NewAppContext(
 		batchEnsemblingJobRunner := batchensembling.NewBatchEnsemblingJobRunner(
 			batchEnsemblingController,
 			ensemblingJobService,
+			ensemblersService,
```

Comment:
The batch job runner now needs the EnsemblersService, to query the python version information at the time of image building. See: `api/turing/batch/ensembling/runner.go`